### ISSUE TITLE
structcheck: noCopy struct is flagged as unused

### ISSUE THREAD

**Issue by egonelbre** [golangci/golangci-lint#1015]
In this code:

```go
package main

type noCopy struct{}

func (noCopy) Lock() {}

type example struct {
	noCopy noCopy
}

func main() {
	_ = &example{}
}
```

noCopy field is flagged as unused by `structcheck`.

```console
$ golangci-lint run
main.go:8:2: `noCopy` is unused (structcheck)
        noCopy noCopy
```

This is the common way of marking structs as they shouldn't be copied. See https://golang.org/issues/8005#issuecomment-190753527 for more information.

<details><summary>Version of golangci-lint</summary>

```console
$ golangci-lint --version
golangci-lint has version 1.24.0 built from 6fd4383 on 2020-03-15T11:38:01Z
```

</details>

**Comment by mkorolyov** [comment]
Hi! any workarounds so far?

**Comment by ernado** [comment]
@mkorolyov 

Something like that:
```yml
issues:
  exclude-rules:
    - text: '`noCopy` is unused'
      linters: [structcheck]
```
